src/safeposix/syscalls/sys_constants.rs

```diff
@@ -3,4 +3,30 @@
 use crate::interface;
 
 // Define constants using static or const
-// Imported into fs_calls file
\ No newline at end of file
+// Imported into fs_calls file
+
+
+//GID AND UID DEFAULT VALUES
+
+DEFAULT_UID : u64 = 1000;
+DEFAULT_GID : u64 = 1000;
+
+
+// RESOURCE LIMITS
+
+NOFILE_CUR : u64 = 1024;
+NOFILE_MAX : u64 = 4*1024;
+
+STACK_CUR : u64 = 8192*1024;
+STACK_MAX : u64 = 2**32;
+
+RLIMIT_STACK = 0;
+RLIMIT_NOFILE = 1;
+
+
+//R Limit for getrlimit system call
+#[repr(C)]
```

Comment:
I don't think we need to make this C typed?

Reply:
We absolutely do because the struct is passed in from C, and we populate its fields in rust

Reply:
Ah ok, I was still in the mindset that we needed to serialize structs like we were doing from Python. I guess this will work much better/easier for structs!